test/cpp/naming/resolver_component_tests_runner_invoker.cc

```diff
@@ -89,6 +92,7 @@ void InvokeResolverComponentTestsRunner(
   gpr_cv test_driver_cv;
   gpr_cv_init(&test_driver_cv);
   int status = test_driver->Join();
+#ifndef GPR_WINDOWS
```

Comment:
nit: while we're here, I think we can do some cleanup:

- I think we can delete `test_driver_mu` and `test_driver_cv`.

- I'm not sure how much value there was in parsing the status and adding a custom log. Instead I think we could just directly return the result of `test_driver->Join();`. Then we could delete the platform ifdefs in this method

Reply:
Done.